docs/clay_over_aoi.ipynb

```diff
@@ -0,0 +1,727 @@
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "! wget https://huggingface.co/made-with-clay/Clay/resolve/main/Clay_v0.1_epoch-24_val-loss-0.46.ckpt?download=true -O data/checkpoints/Clay_v0.1_epoch-24_val-loss-0.46.ckpt"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "46ddcaa2-34d0-44bd-9512-40ed2374f893",
+   "metadata": {},
+   "source": [
+    "After downloading the model weights, embeddings can be created with just one command."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "id": "91f194ad-781c-4fa3-bdd8-6d1af29b23b5",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "W&B disabled.\n",
+      "Seed set to 42\n",
+      "Using 16bit Automatic Mixed Precision (AMP)\n",
+      "GPU available: True (cuda), used: True\n",
+      "TPU available: False, using: 0 TPU cores\n",
+      "IPU available: False, using: 0 IPUs\n",
+      "HPU available: False, using: 0 HPUs\n",
+      "[rank: 0] Seed set to 42\n",
+      "Initializing distributed: GLOBAL_RANK: 0, MEMBER: 1/1\n",
+      "----------------------------------------------------------------------------------------------------\n",
+      "distributed_backend=nccl\n",
+      "All distributed processes registered. Starting with 1 processes\n",
+      "----------------------------------------------------------------------------------------------------\n",
+      "\n",
+      "Total number of chips: 28\n",
+      "Restoring states from the checkpoint path at data/checkpoints/Clay_v0.1_epoch-24_val-loss-0.46.ckpt\n",
+      "LOCAL_RANK: 0 - CUDA_VISIBLE_DEVICES: [0]\n",
+      "Loaded model weights from the checkpoint at data/checkpoints/Clay_v0.1_epoch-24_val-loss-0.46.ckpt\n",
+      "Predicting DataLoader 0: 100%|██████████████████| 14/14 [00:02<00:00,  6.89it/s]Saved 7 rows of embeddings of shape (768,) to /home/tam/Documents/repos/model/data/embeddings/45QUB_20200328_20210825_v001.gpq\n",
+      "Saved 8 rows of embeddings of shape (768,) to /home/tam/Documents/repos/model/data/embeddings/45QTB_20200328_20210815_v001.gpq\n",
+      "Saved 8 rows of embeddings of shape (768,) to /home/tam/Documents/repos/model/data/embeddings/44QRG_20200328_20210815_v001.gpq\n",
+      "Saved 1 rows of embeddings of shape (768,) to /home/tam/Documents/repos/model/data/embeddings/45QVC_20210822_20210822_v001.gpq\n",
+      "Saved 2 rows of embeddings of shape (768,) to /home/tam/Documents/repos/model/data/embeddings/44QRH_20200328_20200328_v001.gpq\n",
+      "Saved 2 rows of embeddings of shape (768,) to /home/tam/Documents/repos/model/data/embeddings/45QTC_20200328_20200328_v001.gpq\n",
+      "Predicting DataLoader 0: 100%|██████████████████| 14/14 [00:02<00:00,  6.77it/s]\n",
+      "Done!\n"
+     ]
+    }
+   ],
+   "source": [
+    "! wandb disabled\n",
+    "! python trainer.py predict --ckpt_path=data/checkpoints/Clay_v0.1_epoch-24_val-loss-0.46.ckpt \\\n",
```

Comment:
Lightning supports passing in a checkpoint URL (see https://lightning.ai/docs/pytorch/2.1.2/common/checkpointing_advanced.html#resume-training-from-a-cloud-checkpoint), so can remove the `wget` line above and predict directly from the HuggingFace checkpoint URL. The checkpoint will be downloaded automatically to `~/.cache/torch/hub/checkpoints/`

```suggestion
    "! python trainer.py predict --ckpt_path=https://huggingface.co/made-with-clay/Clay/resolve/main/Clay_v0.1_epoch-24_val-loss-0.46.ckpt \\\n",
```